src/libs/ReportActionsUtils.ts

```diff
@@ -19,6 +22,15 @@ type LastVisibleMessage = {
     lastMessageHtml?: string;
 };
 
+type MessageActionItemChanelLog = {
```

Comment:
Uh, I'm not sure about this name (`MessageActionItemChanelLog`). I'll try to think about it.